scripts/play.ts

```diff
@@ -0,0 +1,402 @@
+          dirOnly: true,
+        })) ?? createRandomName();
+    }
+  }
+
+  const playgroundDir = new URL(`./${playgroundName}/`, PLAYGROUNDS_ROOT);
+  const playgroundExists = await pathExists(playgroundDir);
+
+  if (playgroundExists) {
+    if (args.new) {
+      throw new CommandError(
+        `A playground named ${playgroundName} already exists. Exiting.`,
+      );
+    }
+
+    if (didProvidePlaygroundName) {
+      console.info(`Using playground: ${chalk.green(playgroundName)}`);
+    } else {
+      console.info(
+        `Reusing most recent playground: ${chalk.green(playgroundName)}`,
+      );
+    }
+
+    console.info('  (Use "--new" to create a new playground)');
+    console.info();
+
+    if (args.framework) {
+      console.warn(
+        chalk.yellow`The --framework/-f argument is ignored when a playground already exists.`,
+      );
+    }
+  } else {
+    console.log(`Creating a new playground: ${chalk.green(playgroundName)}`);
+
+    // Ensure dependencies are isolated from the monorepo.
+    if (!(await pathExists(new URL("./yarn.lock", PLAYGROUNDS_ROOT)))) {
+      await fs.writeFile(new URL("./yarn.lock", PLAYGROUNDS_ROOT), "");
+    }
+
+    await createPlayground(playgroundName, playgroundDir, {
+      framework: args.framework ?? DEFAULT_FRAMEWORK,
+      dryRun: args["dry-run"],
+    });
+  }
+
+  if (args.start) {
+    await exec("yarn", ["dev"], {
+      cwd: playgroundDir,
+      stdio: "inherit",
+      dryRun: args["dry-run"],
+    });
+  }
+
+  process.off("uncaughtException", handleUncaughtException);
+}
+
+/**
+ * Creates a new playground.
+ *
+ * @param name - The name of the playground. It must be unique.
+ * @param dir - The directory of the playground. It must be unique.
+ * @param options - Options that determine the function's behavior.
+ */
+async function createPlayground(
+  name: string,
+  dir: URL,
+  options: DryRunOption & {
+    /**
+     * The framework used to bootstrap the playground.
+     */
+    framework: Args["framework"];
+  },
+) {
+  switch (options.framework ?? "next") {
+    case "next": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nextjs-starter-prismic-minimal-ts.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      await updatePackageJSON(
+        dir,
+        { scripts: { "next:dev": "next dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      await updateSliceMachineConfig(
+        dir,
+        { localSliceSimulatorURL: "http://localhost:8001/slice-simulator" },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "sveltekit": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/sveltekit-starter-prismic-minimal.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    default: {
+      console.error(chalk.red`Framework not supported: ${options.framework}`);
```

Comment:
We don't support Nuxt today with this script? What do we miss?

Reply:
We do now!